packages/usa-dark-background/src/styles/_usa-dark-background.scss

```diff
@@ -9,14 +11,10 @@
   }
 
   a {
-    color: color($theme-link-reverse-color);
-
-    &:visited {
-      color: color($theme-link-reverse-color);
-    }
-
-    &:hover {
-      color: color($theme-link-reverse-hover-color);
-    }
+    @include set-link-from-bg(
+      "base-darker",
+      $theme-link-reverse-color,
+      $context: $background-context
+    );
```

Comment:
@amyleadem good find. Yes, please create an issue.

Reply:
Opened https://github.com/uswds/uswds/issues/5659 for follow-up

Reply:
I believe the reason why this ignores `active` is to allow the original link color to be lighter (since there are fewer lightening steps to account for without `active`).